app/models/builders/components.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
Is this file needed?

Reply:
Most definately, this declares the full interface for govukCheckbox - going to make life much easier when writing the model for components